eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -239,6 +239,11 @@ jobs:
         displayName: 'Publish Report Artifacts'
         artifact: reports
 
+      - publish: $(System.DefaultWorkingDirectory)
```

Comment:
Should this be `Pipeline.Workspace` instead? I'm not sure if the default working directory can be changed or not.